GADDirectory/GADQuery.h

```diff
@@ -0,0 +1,46 @@
+//
+//  GADQuery.h
+//  DB
+//
+//  Created by Liam li on 3/5/17.
+//  Copyright © 2017 AppDev. All rights reserved.
+//
```

Comment:
Please remove these header comments. They are unnecessary, redundant, and inaccurate.